apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -659,11 +661,19 @@ const BookingPage = ({
                       ? "-mt-4"
                       : ""
                   )}>
+                <Turnstile
+                    className="fixed bottom-4 right-4"
+                    onError={() => setStatus("error")}
```

Comment:
It looks like we're not using the error and expire success states

Reply:
Ok, @baileypumfleet  I will change these and I have to make a new PR right ? Also I used lint:fix and that fixed all linting errors, I think this the right way to do it. Sorry for the silly questions actually I am new to open source contributions.